OrbitCore/Path.cpp

```diff
@@ -282,6 +282,12 @@ std::string Path::GetHome() {
   return home;
 }
 
+std::string Path::GetLogFilePath() {
+  std::string logsDir = Path::GetAppDataPath() + "logs/";
```

Comment:
nit: use JoinPath() here and below - let it take care of '/' separators

Reply:
Added JoinPath() instead of separator